src/phases/select_action_phase.js

```diff
@@ -7,13 +7,31 @@ This Source Code Form is subject to the terms of the Mozilla Public License, v.
 This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
 */
 
-const selectActionPhase = (actions, selectActionDialog, insufficientFundsDialog, genericErrorDialog) => async () => {
-  const actionSelectionList = Object.keys(actions);
+import {APOLLO, APOLLO_CODE, ATLAS_1, ATLAS_2, ATLAS_3, ATLAS_CODE, HERMES, HERMES_CODE, NO_ROLE_CODE} from '../consts';
+
+const getTypeCode = (nodeRole) => {
```

Comment:
We have very similar method in `smart_contracts_model`, with reverse effect, called `roleCodeToRole(roleCode, deposit)`.
Maybe we should name them analogically (sth like `roleCodeToRole` and `roleToRoleCode`) and move them to one place. Sth like roles utils?
Them we may move there also this:
```
    if (role === HERMES) {
      nodeTypeName = 'hermes';
    } else if (role === APOLLO) {
      nodeTypeName = 'apollo';
    } else if (role === ATLAS_1 || role === ATLAS_2 || role === ATLAS_3) {
      nodeTypeName = 'atlas';
    } else {
      throw new Error('Invalid role');
    }
```
and also this
`atlasStakeAmountToRole(deposit)`

Generally maybe a good idea is to collect in one place all the role-roleCode-roleName-deposit-whatever translations.

Reply:
Generally, good idea, but requires quite a big refactoring and I'd prefer to do it in another PR

Reply:
Actually, it wasn't that big :)